Filters/exclusions.txt

```diff
@@ -1,3 +1,5 @@
+! https://github.com/AdguardTeam/FiltersRegistry/issues/186
+||adswizz.com^
```

Comment:
I suggest adding NPR domains to exceptions instead:
`@@||npr.streaming.adswizz.com^`
`@@||cf-npr.streaming.adswizz.com^`

Reply:
Well, nevermind, it seems that bloccking the whole domain is quite problematicc